style.css

```diff
@@ -3,12 +3,9 @@
   width: 100%;
 }
 
-.searchBox {
-  color: white;
-}
-
 #search {
-  padding-left: 10px;
```

Comment:
Why did you remove this? The padding allowed the text to not be right against the search box border

Reply:
Sorry about this, I noticed I did something to make it right against the search, but I did not know what!